### DISCUSSION TITLE
Auto Restart on fail option

### DISCUSSION THREAD

**Discussion by Otter-Lover** [monkeytypegame/monkeytype#1805] [Ideas] [answered]
Maybe an option that you can enable that auto restarts if you fail so you don't got to keep clicking to start again.

**Comment by Miodec** [comment, marked as answer]
Already asked for many times. I don't want to do things for the user. Especially things like restarting the test automatically. Because then there will be situations where they fail many tests in quick succession because they didn't realize they failed.

If by 'keep clicking' you mean using the mouse, try using tab + enter, or just tab with quick tab mode enabled in the settings.

**Reply by Otter-Lover** [reply]
True, You could do something that pops up on the side that says you failed or something.
